### DISCUSSION TITLE
1.1.2.3

### DISCUSSION THREAD

**Discussion by ilyhalight** [ilyhalight/voice-over-translation#143] [Обновления]
Вроде бы, всё работает и ничего не сломалось после рефакторинга. Пробуйте-тестируйте.

- Добавлена поддержка rutube embed
- Еще один рефакторинг от @SashaXse ([ilyhalight/voice-over-translation#142](https://github.com/ilyhalight/voice-over-translation/pull/142)

## What's Changed
* Refractoring #2 (Молюсь аллаху что я нечего не сломал) by @SashaXser in https://github.com/ilyhalight/voice-over-translation/pull/142

**Full Changelog**: https://github.com/ilyhalight/voice-over-translation/compare/1.1.2.2...1.1.2.3

<hr /><em>This discussion was created from the release <a href='https://github.com/ilyhalight/voice-over-translation/releases/tag/1.1.2.3'>1.1.2.3</a>.</em>

**Comment by FiLinX** [comment]
а ютубные эмбеды уже не вернутся?  помойму уже несколько версий как на них нету панельки даже..

**Reply by ilyhalight** [reply]
позже пофикшу
